EIPS/eip-4521.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 4521
+title: 721/20-compatible transfer
+description: Recommends a simple extension to make NFTs compatible with apps and contracts that handle fungibles.
+author: Ross Campbell (@z0r0z)
+discussions-to: https://github.com/ethereum/EIPs/issues/4521
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-12-13
+requires: 721
+---
+
+## Abstract
+ERC-721, the popular standard for non-fungible tokens (NFTs), includes send functions, such as `transferFrom()` and `safeTransferFrom()`, but does not include a backwards-compatible `transfer()` found in fungible ERC-20 tokens. This standard provides references to add such a `transfer()`.
+
+## Motivation
+This standard proposes a simple extension to allow NFTs to work with contracts designed to manage ERC-20s and many consumer wallets which expect to be able to execute a token `transfer()`. For example, if an NFT is inadvertently sent to a contract that typically handles ERC-20, that NFT will be locked. It should also simplify the task for contract programmers if they can rely on `transfer()` to both handle ERC-20 and NFTs.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+The interface for ERC-4521 `transfer()` MUST conform to ERC-20 and resulting transfers MUST fire the `Transfer` event as described in ERC-721.
+
+```sol
+function transfer(address to, uint256 tokenId) external returns (bool success);
+```
+
+## Rationale
+Replicating ERC-20 `transfer()` with just a minor change to accurately reflect that a unique `tokenId` rather than fungible sum is being sent is desirable for code simplicity and to make integration easier.
+
+## Backwards Compatibility
+There are no production instances of something like ERC-4521 that the author is aware of, but an implementation is [in review](https://github.com/Rari-Capital/solmate/pull/63) for inclusion in RariCapital’s SolMate library.
+
+## Reference Implementation
+A reference contract for an ERC-4521 NFT can be found [here](https://github.com/z0r0z/EIP-4521/blob/main/contracts/ERC4521.sol).
```

Comment:
Don't include external links.  Instead, either inline the contract here or add it as an asset to `../assets/eip-4521/` and do a relative link to it here.

Reply:
thanks - I made this update.